packages/patterns/src/keys/compareKeys.js

```diff
@@ -147,6 +147,30 @@ export const compareKeys = (left, right) => {
       // @ts-expect-error narrowed
       return compareRank(left.length, right.length);
     }
+    case 'byteArray': {
```

Comment:
Given the non-recursive nature of pass style "byteArray", I wonder if it belongs earlier in these `switch` cases. But we're not fully consistent anyway, so no harm if you want to leave it after "copyArray" (which is recursive).